config/multiple-workers/orderly-web.yml

```diff
@@ -0,0 +1,122 @@
+## Prefix for container names; we'll use {container_prefix}_(orderly,web)
```

Comment:
you should be able to avoid this by modifying the above config at the point of tests (they accept a dictionary of overrides - there will be other examples in the tests). That will reduce keeping things in sync in future

Reply:
Thanks - tidied that up